int32  actionType
int32  motorNumber
int32  control
int32  speed
string comment
